.eslintrc.js

```diff
@@ -18,33 +14,39 @@ module.exports = {
         "one-var": ["warn"],
         "padded-blocks": ["warn"],
         "no-extend-native": ["warn"],
-        "camelcase": ["warn"],
-        "no-multi-spaces": ["error", { "ignoreEOLComments": true }],
-        "space-before-function-paren": ["error", {
-            "anonymous": "never",
-            "named": "never",
-            "asyncArrow": "always",
-        }],
+        camelcase: ["warn"],
+        "no-multi-spaces": ["error", { ignoreEOLComments: true }],
+        "space-before-function-paren": [
+            "error",
+            {
+                anonymous: "never",
+                named: "never",
+                asyncArrow: "always",
+            },
+        ],
         "arrow-parens": "off",
         "prefer-promise-reject-errors": "off",
-        "quotes": "off",
-        "indent": "off",
+        quotes: "off",
+        indent: "off",
         "no-constant-condition": "off",
         "no-async-promise-executor": "off",
     },
-    overrides: [{
-        "files": ["src/**/*.ts", "test/**/*.ts"],
-        "extends": ["plugin:matrix-org/typescript"],
-        "rules": {
-            // TypeScript has its own version of this
-            "babel/no-invalid-this": "off",
+    overrides: [
+        {
+            files: ["src/**/*.ts", "test/**/*.ts"],
+            extends: ["plugin:matrix-org/typescript", "prettier"],
```

Comment:
What does adding `"prettier"` to `extends` do?  I think it's not needed because the rules being changed are already pulled in by `matrix-org/typescript`.